### PR TITLE
task html layout form

### PR DESCRIPTION
**Pull request by YuriyLarin**
<a href="https://YuriyLarin.github.io/layout_html-form/">DEMO LINK</a>
<a href="https://YuriyLarin.github.io/layout_html-form/report/html_report/">TEST REPORT LINK</a>

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,186 @@
...
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="input">
+          <label for="email" >E-mail:</label>
```
Comment (loralevitska) [inline src/index.html]: Check all your work and fix all cases if present
```suggestion
          <label for="email">E-mail:</label>
```

**Review by loralevitska — CHANGES_REQUESTED** [review]
Add demo_link: deploy your solution, and before review check demo_link.  
image

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,186 @@
...
+
+        <div>
+          <label for="pass">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="pass"
+            minlength="6"
+          >
+        </div>
```
Comment (SanyaBratashchuk) [inline src/index.html:94]: get rid of this div's. Apply `display:block` to the labels instead

File: src/style.css [hunk 3]
```diff
@@ -1 +1,7 @@
-/* styles go here */
+.input {
```
Comment (SanyaBratashchuk) [inline src/style.css:1]: apply this class to each field but change selector to `.input:not(:last-child)`

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (YuriyLarin) [reply src/style.css:1]: Are you kidding me? I did that in the first place. But the mentor did not approve of such a solutionand said: just wrap everything in a <div>.  I have been doing this task for more than two mounts now. One says one thing, the other another. Do you have common standarts?

**Pull request closed without merging**